MediaSession.cpp

```diff
@@ -73,10 +73,14 @@ const DRM_WCHAR g_rgwchCDMDrmStoreName[] = {WCHAR_CAST('/'), WCHAR_CAST('t'), WC
 const DRM_CONST_STRING g_dstrCDMDrmStoreName = CREATE_DRM_STRING(g_rgwchCDMDrmStoreName);
 
 #ifdef PR_3_3
+#ifdef NETFLIX
 const DRM_CONST_STRING *g_rgpdstrRights[1] = {&g_dstrDRM_RIGHT_PLAYBACK};
 #else
 const DRM_CONST_STRING *g_rgpdstrRights[1] = {&g_dstrWMDRM_RIGHT_PLAYBACK};
 #endif
+#else
+const DRM_CONST_STRING *g_rgpdstrRights[1] = {&g_dstrWMDRM_RIGHT_PLAYBACK};
+#endif
```

Comment:
Could you elaborate on this please? I get 2.5 vs 3.3 API change issue or using API wrapped with NETLIX_EXT but do not see reason behind this. For me g_dstrWMDRM_RIGHT_PLAYBACK should always be used.

Reply:
I don't have an idea in which version g_dstrDRM_RIGHT_PLAYBACK comes. Thinking that NETFLIX extension may use. So it seems there is no case this definition exist and g_rgdpstrRights can be defined unconditionally